css/css-masking/clip-path/clip-path-inline-001.html

```diff
@@ -4,6 +4,7 @@
 <link rel="help" href="https://www.w3.org/TR/css-masking-1/#the-clip-path" title="5.1 Clipping Shape: the clip-path property">
 <link rel="match" href="../../reference/ref-filled-green-100px-square.xht">
 <meta content="ahem" name="flags">
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css" />
```

Comment:
This test has `color: transparent` so the text isn't there. To debug it I'd start by removing that to see if the font size appears to have changed, or maybe we're not even loading the webfont?

In general, webfonts aren't loaded until they're found to be needed by layout, so I'd look out for bugs around that in Chrome.

Reply:
The general behaviour when the font isn't loaded is that you see the letter X on the page in the tests. When the font is loaded, the X renders as a square.

For this test (and all tests in this PR actually), I never see X in the screenshots, it's always boxes. But they're sometimes red or not in the right arrangement. So I'd assume it's not a matter of timing?